core/src/main/java/org/openconnectors/util/SourceConnectorContext.java

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Whats SourceContext and SourceConnectorContext?

Reply:
you can understand purpose as per this example.

https://github.com/openconnectors/connectors-legacy-prototype/blob/master/java/core/src/main/java/org/streamlio/localfs/LocalFSSource.java